server/mergin/sync/permissions.py

```diff
@@ -239,3 +239,27 @@ def get_user_project_role(project: Project, user: User) -> Optional[str]:
         return "writer"
     if ProjectPermissions.Read.check(project, user):
         return "reader"
+
+
+def project_admin_permissions(workspace_id, user: User) -> bool:
```

Comment:
I wonder if we need this in CE at all since this is kind of workspace permission only for a particular use case in EE